optuna/cli.py

```diff
@@ -70,16 +68,38 @@ def get_parser(self, prog_name: str) -> ArgumentParser:
             help="If specified, the creation of the study is skipped "
             "without any error when the study name is duplicated.",
         )
+        parser.add_argument(
+            "--directions",
```

Comment:
Not a strong opinion but how about using the available `nargs` instead (in which case you don't comma separate and split but receive a list when separated by whitespace)?

```suggestion
            "--directions",
            nargs="+",
```

Reply:
Thank you so much! I didn't know about `nargs`. I revised the PR to use `nargs` instead of a comma-separated text in 98ddb39.